### ISSUE TITLE
[BERT/Pytorch] Please list the results of downtown tasks for the pretraining bert.

### ISSUE THREAD

**Issue by mx8435** [NVIDIA/DeepLearningExamples#1133]
Related to **Model/Framework(s) or something else (describe)** 

*Examples:* 
* *PyTorch* 
* *AMP* 

**Describe the solution you'd like**
Now only the training loss is shown in `README.md`, please list the GLUE results of the BERT pretrained by `Pytorch/LanguageModel/BERT/scripts/run_pretrain.sh`.

**Comment by sharathts** [comment]
Do you mean loss curve or accuracy results? Accuracy results on fine-tuning tasks are published here: https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/LanguageModeling/BERT#fine-tuning-accuracy-results-nvidia-dgx-a100-8x-a100-80gb
